### PR TITLE
feat(pt/dpmodel): support type_one_side in se_e2_a

### PR DESCRIPTION
**Pull request by njzjz**

### CODE REVIEW

File: deepmd/dpmodel/descriptor/se_e2_a.py [hunk 1]
```diff
@@ -313,6 +327,17 @@ def call(
...
+            for embedding_idx in itertools.product(range(self.ntypes), repeat=2):
+                # not actually used; to match serilization data from TF to pass the test
+                if embedding_idx in self.emask:
+                    for ilayer in range(len(self.neuron)):
+                        layer = self.embeddings[embedding_idx][ilayer]
+                        layer.w.fill(0.0)
+                        layer.b.fill(0.0)
+                        if layer.idt is not None:
+                            layer.idt.fill(0.0)
+
```
Comment (wanghan-iapcm) [inline deepmd/dpmodel/descriptor/se_e2_a.py:340]: ugly code. can tf avoids such serialization?

**Review by wanghan-iapcm — APPROVED** [review]

File: deepmd/dpmodel/descriptor/se_e2_a.py (hunk 1 above)
Comment (wanghan-iapcm) [reply deepmd/dpmodel/descriptor/se_e2_a.py:340]: or the hacking code should be moved to network collection?

Comment (wanghan-iapcm) [reply deepmd/dpmodel/descriptor/se_e2_a.py:340]: why we did not see the hack in pt...

Comment (njzjz) [reply deepmd/dpmodel/descriptor/se_e2_a.py:340]: PT doesn't compare with TF directly.

Comment (njzjz) [reply deepmd/dpmodel/descriptor/se_e2_a.py:340]: DP and PT will store this network, although it is never used. TF will lose the information after deserializing + serializing.

Comment (njzjz) [reply deepmd/dpmodel/descriptor/se_e2_a.py]: I move it to `Network.clear()`. See b3160c878aaed8187cc60589bdcd70f7be872dda.

**Pull request merged**